src/Persistence/MassTransit.Azure.Cosmos/MassTransit.Azure.Cosmos.csproj

```diff
@@ -25,6 +25,7 @@
     <PackageReference Include="Microsoft.SourceLink.GitHub" PrivateAssets="All" />
     <ProjectReference Include="..\..\MassTransit\MassTransit.csproj" />
     <PackageReference Include="Microsoft.Azure.Cosmos" />
+    <PackageReference Include="Microsoft.Extensions.Options" />
```

Comment:
do you really need it? I think it is already part of MT dependencies

Reply:
It is part of MT dependencies 
https://github.com/MassTransit/MassTransit/blob/3db6cafad9a536a45648eeb79bbd8d6a2e424ff0/Directory.Packages.props#L55
but not referenced in this projet.

Reply:
We are using options extensively, so it should just work when you reference `MassTransit.csproj`, so you can remove it

Reply:
Right! Dependency removed. Thank you